phpstan-baseline.neon

```diff
@@ -1,5 +1,15 @@
 parameters:
 	ignoreErrors:
+		-
```

Comment:
Ahhh... so we're taking the easy way out? 🙃

Reply:
For now :)